.gitignore

```diff
@@ -5,3 +5,9 @@ bazel-*
 *.egg-info
 ignore/*
 tutorials/.ipynb_checkpoints/
+tutorials/trash
+native/results
+native/result
```

Comment:
Can we just merge `native/results` and `native/result`?

Reply:
These will get cleaned out soon.

Reply:
Fixed.